atlasdb-client/src/main/java/com/palantir/atlasdb/schema/cleanup/ArbitraryCleanupMetadata.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.schema.cleanup;
+
+import com.google.common.base.Throwables;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.palantir.atlasdb.protos.generated.SchemaMetadataPersistence;
+
+public class ArbitraryCleanupMetadata implements CleanupMetadata {
+    public static Hydrator<ArbitraryCleanupMetadata> BYTES_HYDRATOR = message -> {
+        try {
+            return hydrateFromProto(SchemaMetadataPersistence.ArbitraryCleanupMetadata.parseFrom(message));
+        } catch (InvalidProtocolBufferException e) {
+            throw Throwables.propagate(e);
+        }
+    };
+
+    public static ArbitraryCleanupMetadata hydrateFromProto(
+            SchemaMetadataPersistence.ArbitraryCleanupMetadata unused) {
+        // For now, this is correct!
```

Comment:
...when will it not be? When that happens, will this comment become amusingly out of date?

Reply:
Fair enough - I'll rewrite this as `this is correct while the ArbitraryCleanupMetadata message has no fields`, which is what it is

Reply:
There's also probably the same message for the Null one, as well